### PR TITLE
making copyedits and adding feedback comments

### PR DESCRIPTION
**Pull request by osisoft-mbishop**

### CODE REVIEW

File: content/administration/delete-configuration.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ edgecmd -port 5591 <RestOfTheCommand>
 
 ## Delete configuration entry
 
-Complete the following steps to delete a configuration entry from a collection configuration. For example, you can delete a single health endpoint of the 'HealthEndpoints' facet within the 'System' component.
+Complete the following steps to delete a configuration entry from a collection configuration. For example, you can delete a single health endpoint of the `HealthEndpoints` facet within the `System` component.
```
Comment (osisoft-mbishop) [inline content/administration/delete-configuration.md:17]: Throughout the doc, I noticed some use of single quotes. I would advise changing them to backticks or double quotes, although I couldn't find a clear style guide rule about this.

File: content/configuration/configure-adapter-data-selection-with-csv-files.md [hunk 2]
```diff
@@ -2,28 +2,28 @@
 uid: ConfigureAdapterDataSelectionWithCSVFiles
 ---
 
-# Configure adapter data selection with CSV files
+# Configure adapter data selection with .csv files
```
Comment (osisoft-mbishop) [inline content/configuration/configure-adapter-data-selection-with-csv-files.md:5]: [MS style guide](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions) wants us to use all lowercases for file extensions.

File: content/installation/install-edgecmd-utility.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ Complete the following steps to install EdgeCmd utility on Windows:
 
 2. Run the .msi file.
 
-   **Note:** To change the install path from the default path of *C:\Program Files\OSIsoft\EdgeCmd*, enter the following command in the command prompt and update the <file_path>. OSIsoft recommends you use the default value.
+   **Note:** To change the install path from the default path of *C:\Program Files\OSIsoft\EdgeCmd*, enter the following command in the command prompt and update the *<file_path>*. OSIsoft recommends you use the default value.
```
Comment (osisoft-mbishop) [inline content/installation/install-edgecmd-utility.md:23]: Style guide wants variables in italics.

File: content/installation/install-edgecmd-utility.md [hunk 4]
```diff
@@ -62,7 +62,7 @@ Complete the following steps to install EdgeCmd utility on Linux:
 
 2. After the check for prerequisites succeeds, accept the default port number `5590` or change the port number.
 
-    If the Linux OS is up to date, the install will succeed and EdgeCmd utility is running on your device.
+    If the Linux OS is up-to-date, the install succeeds and EdgeCmd utility starts running on your device.
```
Comment (osisoft-mbishop) [inline content/installation/install-edgecmd-utility.md:65]: [Up-to-date](https://ahdictionary.com/word/search.html?q=up+to+date) is hyphenated. Also edited the sentence as it read awkwardly.

File: content/troubleshooting/troubleshoot-edgecmd-utility.md [hunk 5]
```diff
@@ -13,5 +13,5 @@ To troubleshoot issues with EdgeCmd utility, you can start with the following tr
     - If you do not know what port to use, check the `appsettings.json` file in the directory where the adapter is installed. 
 
 3. Check if the adapter is running.<br>EdgeCmd utility makes REST calls against an active adapter. If the adapter is not running, there is nothing to respond to those REST requests.
-4. If you run a command like `Reset`, add a data source configuration, or edit a large data selection, and so on. Wait 20 to 60 seconds before you attempt to make more calls; the adapter needs time to process the request.
+4. If you run a command like `Reset`, add a data source configuration, edit a large data selection, or so on. Wait 20 to 60 seconds before you attempt to make more calls; the adapter needs time to process the request.
```
Comment (osisoft-mbishop) [inline content/troubleshooting/troubleshoot-edgecmd-utility.md:16]: Tweaked this sentence, as it read awkwardly.

**Review by osisoft-mbishop — COMMENTED** [review]
Hi Tina,

I noticed a few style issues, but nothing that should hold up publishing. I also left some inline comments that we should probably address at some point.

File: content/administration/delete-configuration.md (hunk 1 above)
Comment (TinaUll) [reply content/administration/delete-configuration.md:17]: @osisoft-mbishop Thanks! Those should definitely backticks.

File: content/installation/access-edgecmd-utility.md [hunk 6]
```diff
@@ -15,6 +15,8 @@ Complete the following steps to access EdgeCmd utility on Windows:
 
    **Note:** EdgeCmd is added to PATH, so you can execute it from any folder.
 
+   <!-- Mark Bishop 9/16: Not sure if I encountered a bug, but I ran through install and EdgeCmd DID NOT automatically get added to PATH. I had to either run from the install path or manually add the install path to PATH. -->
```
Comment (TinaUll) [inline content/installation/access-edgecmd-utility.md:18]: @osisoft-mbishop, hmm, maybe you used an old build? Works for me.

**Pull request merged**